doc_conf/conf.py

```diff
@@ -51,25 +42,35 @@
     "sphinx.ext.mathjax",
     "sphinx_gallery.gen_gallery",
     "numpydoc",
+    "sphinx.ext.linkcode",
 ]
 
-if LooseVersion(sphinx_gallery.__version__) < LooseVersion("0.2"):
-    raise ImportError(
-        "Must have at least version 0.2 of sphinx-gallery, got "
-        "%s" % (sphinx_gallery.__version__,)
-    )
+templates_path = ["_templates"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
-matplotlib.use("agg")
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = "en"
 
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates"]
+html_static_path = ["_static"]
+
+
+html_logo = "_static/logo.png"
+html_theme = "pydata_sphinx_theme"
+html_title = "HiDimStat"
+
+plot_formats = []
```

Comment:
Why do you define `plot_formats` as empty list?

Reply:
If you need to keep it, you should move it to the section of the options for the gallery.

Reply:
I guess it's not needed. Also, it's not in Skrub's conf file.